ui/app/styles/directives.css

```diff
@@ -120,4 +120,24 @@
     display: inline-block;
     padding-left: 10px;
     min-width: 150px;
+}
+
+
+/**
+ * product-import-tree
+ */
+.bc-tree .node-sucess {
+    color: #3f9c35;
+}
+.bc-tree .node-warning {
+    color: #ec7a08;
+}
+.bc-tree .node-critical {
+    color: #cc0000;
+}
+
+.bc-tree .node-selected .node-sucess,
```

Comment:
I'm just wondering, could we use the patternfly built in css classes ".text-success .text-warning .text-danger" rather than custom css classes? This would ensure we match up with the patternfly colour scheme if it changes slightly for example.